### DISCUSSION TITLE
custom fields for tasks

### DISCUSSION THREAD

**Discussion by niels-przybilla** [kimai/kimai#4785] [General] [answered]
Hello Kimai Team,

is it possible to add a custom field to a task (in our case a ticket number)

Many thanks and best wishes

Niels

**Comment by kevinpapst** [comment, marked as answer]
Hi Niels, 

Not yet, but I guess there is a feature request already existing, where you can add your vote.

I am closing here, as this repository is about Kimai itself.

Please post request for paid plugins at https://github.com/kevinpapst/kimai-plugins next time.
